### PR TITLE
Add gever_url field to workspaces, documents and mails

### PR DESCRIPTION
**Pull request by tinagerber**
We add a field `gever_url` to workspaces and documents. If a workspace is linked to a dossier, `gever_url` refers to the linked dossier. If a workspace document is linked to GEVER document, `gever_url` refers to the linked GEVER document.

For [CA-3707]

## Checklist

_Everything has to be done/checked. Checked but not present means the author deemed it unnecessary._

- [x] Changelog entry
- [x] Link to issue (Jira or GitHub) and backlink in issue (Jira)

_Only applicable should be left and checked._

- New functionality:
  - [x] for `document` also works for `mail`
- New translations
  - [x] All msg-strings are unicode

[CA-3707]: https://4teamwork.atlassian.net/browse/CA-3707?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: opengever/workspaceclient/linked_workspaces.py [hunk 1]
```diff
@@ -226,6 +226,7 @@ def copy_document_to_workspace(self, document, workspace_uid, lock=False):
 
         file_ = document.get_file()
         document_repr = self._serialized_document_schema_fields(document)
+        document_repr['gever_url'] = self.client.get_gever_url(Oguid.for_object(document).id)
```
Comment (njohner) [inline opengever/workspaceclient/linked_workspaces.py:229]: That seems wrong to me. If there is no file, we do not link the document, but you set the `gever_url`

**Review by njohner — COMMENTED** [review]
See comment, otherwise LGTM

**Comment by tinagerber** [comment]
@njohner Can you look at that one again too?

**Review by njohner — APPROVED** [review]
LGTM

**Pull request merged**